build.gradle

```diff
@@ -0,0 +1,28 @@
+plugins {
```

Comment:
What is plugin in gradle build ?

Reply:
In Gradle, a plugin is a piece of code that extends or enhances the functionality of the build tool. Plugins are used to add tasks, manage dependencies, configure build scripts, and integrate external tools and services into the build process.